test/e2e/apps/statefulset.go

```diff
@@ -17,13 +17,18 @@ limitations under the License.
 package apps
 
 import (
+	"encoding/json"
```

Comment:
Use `"k8s.io/apimachinery/pkg/util/json"` instead because it preserves k8s data type conventions (e.g. int instead of float).